IdentityCore/src/webview/embeddedWebview/MSIDOAuth2EmbeddedWebviewController.m

```diff
@@ -377,6 +377,22 @@ - (void)decidePolicyForNavigationAction:(WKNavigationAction *)navigationAction
         return;
     }
     
+    // Handle anchor links that were clicked
+    if([navigationAction navigationType] == WKNavigationTypeLinkActivated) {
+#if TARGET_OS_IPHONE
+        UIApplication *application = [UIApplication performSelector:@selector(sharedApplication)];
```

Comment:
I think this code should be moved down to line 390, since if we are not opening the URL, no need to get an instance of the application.

Reply:
Removed use of application here and used MSIDAppExtensionUtil instead.